1.10/installing/custom/system-requirements/install-docker-centos.md

```diff
@@ -19,6 +19,7 @@ In addition to the general [Docker requirements and recommendations for DC/OS][1
   ```bash
   mkfs -t xfs -n ftype=1 /dev/sdc1
   ```
+*  If using Docker 1.12 or Docker 1.13, then a flag `MountFlags=private` needs to be added in `/etc/systemd/system/docker.service.d/execstart.conf`.  Please do a `systemctl daemon-reload ; systemctl restart docker` after updating this config.
```

Comment:
Suggest revise to: 


If you are using Docker 1.12 or Docker 1.13, you must add `MountFlags=private` in `/etc/systemd/system/docker.service.d/execstart.conf` and then restart the Docker daemon with this command:

  ```bash
  systemctl daemon-reload; systemctl restart docker
  ```